lib/mayaUsd/ufe/UsdTransform3dCommonAPI.cpp

```diff
@@ -64,6 +66,8 @@ class CommonAPITranslateUndoableCmd : public UsdSetXformOpUndoableCommandBase
 
     bool set(double x, double y, double z) override
     {
+        UsdUfe::OperationEditRouterContext editContext(
+            UsdUfe::EditRoutingTokens->RouteTransform, getPrim());
```

Comment:
Thanks for doing this. I think having a unified callback for all transform types has been a good decision. If the need to expose them separately rises, we could always add individual ones.